plugins/chat/app/models/chat/channel.rb

```diff
@@ -39,12 +39,16 @@ class Channel < ActiveRecord::Base
     validate :ensure_slug_ok, if: :slug_changed?
     before_validation :generate_auto_slug
 
+    scope :optional_categories,
```

Comment:
Not sure I get the meaning of "optional_categories" here

Reply:
I didn’t find a better name for this scope which is a left outer join so making the join “optional” in a sense. (it joins on categories but will work on DM that don’t have categories). Glad to change it if you have a better idea :grin:

Reply:
What about renaming it to just `with_categories` maybe?

Reply:
Yes that's was my best bet so far